descriptors/ModuleDescriptor-template.json

```
@@ -2,6 +2,23 @@
   "id": "${artifactId}-${version}",
   "name": "Inventory Module",
   "provides": [
+    {
+      "id": "inventory-batch",
+      "version": "0.1",
+      "handlers": [
+        {
+          "methods": ["POST"],
+          "pathPattern": "/inventory/instances/batch",
+          "permissionsRequired": ["inventory.instances.item.post.batch"],
```

We've used different conventions for specialist endpoints like this. Please only take this as a alternative suggestion, if you prefer what you already have please just resolve without changing

```suggestion
          "permissionsRequired": ["inventory.instances.batch.post"],
```